plugins/ruby-foreman-azure/debian/install

```diff
@@ -0,0 +1,2 @@
+foreman_azure.rb  usr/share/foreman/bundler.d
+cache                usr/share/foreman/vendor
```

Comment:
also needed here for assets:

foreman_azure                 var/lib/foreman/public/assets